SConstruct

```diff
@@ -631,6 +630,34 @@ def add_cluster_analysis(w):
             "seqmagick convert --include-from-file $SOURCES - | " +
             "seqmagick convert --squeeze - $TARGET")
 
+    if options['write_pruned_partis_outfile']:
+        @w.add_target()
+        def pruned_partis_outfile(outdir, c):
+            #these are not the unique ids we want to use when subsettig the cluster, they are just a way to identify the cluster we want to subset
+            clust_ids_string = c['cluster']['unique_ids']
+            yaml_format = partisutils.getsuffix(c['partition']['partition-file']) == '.yaml'
+            return env.Command(
+                path.join(outdir, 'pruned_partis_output.yaml'),
+                [c['partition']['partition-file'], c['pruned_ids']],
+                'python bin/write_subset_partis_outfile.py $SOURCES $TARGET' +
+                ' --partition-step={}'.format(c['partition']['step']) +
+                ' --original-cluster-unique-ids={}'.format(clust_ids_string) + 
+                ' --sw-cache={}'.format(c['sample']['sw-cache']) + 
+                (' --glfo-dir={}'.format(c['sample']['glfo-dir']) if not yaml_format else '') +
+                (' --locus={}'.format(locus(c)) if not yaml_format else '') )
+        
+        @w.add_target()
+        def linearham_base_command(outdir, c):
+            if 'seed' in c:
+                print(path.join(os.getcwd(), str(c['pruned_partis_outfile'][0])))
+                return env.Command(
+                    path.join(outdir, 'linearham_base_command.txt'),
+                    c['pruned_partis_outfile'], #get cwd for absolute path
+                    'echo "scons --run-linearham --template-path=templates/revbayes_template.rev ' +
```

Comment:
Can you please add a little context (in comment or whatever) about why you're writing this command out to a file, and what the overall exec strategy is here?

Reply:
Yes, will do. @matsen stated the goal if possible of running linearham for seeded clusters directly from the cft pipeline, so before we go all the way with that, this is the next best thing: 

The idea is that we can copy this command (minimum necessary to run LH) and run it in our Linearham environment, instead of having to manually build up the command from the parameter dirs, pruned cluster sequences partis yaml file, etc.

Reply:
Yup; That all makes sense. Thanks!